app-network/ucx/autobuild/defines

```diff
@@ -0,0 +1,24 @@
+PKGNAME=ucx
+PKGSEC=science
+PKGDEP="gcc glibc libtool pkg-config rdma-core fuse-3"
+BUILDDEP="autoconf automake m4"
+BUILDDEP__AMD64="${BUILDDEP} cuda valgrind"
+BUILDDEP__ARM64="${BUILDDEP} cuda valgrind"
+PKGDES="Unified Communication Library"
```

Comment:
Best to be a little more descriptive, what does it do? What does it contain?

Reply:
It's a unified library for communication, I can't find description for it.

- 'Unified' means it can support various hardware from different vendors with unified abstraction.
- 'Communication' means it is related to communication.
- 'Library' means other apps can use it as a library.